app/src/main/java/com/example/splendormobilegame/model/Game.java

```diff
@@ -109,5 +111,9 @@ public void setWhosTurn(User whosTurn) {
         this.whosTurn = whosTurn;
     }
 
+    public static ArrayList<EndTurn.PlayerDataResponse> getPlayerRanking(){ return playerRanking; }
+
+    public static void setPlayerRanking(ArrayList<EndTurn.PlayerDataResponse> currentPlayerRanking){ playerRanking = currentPlayerRanking; }
```

Comment:
Class `Model` should be the only class with static fields. These fields in the `Game` shouldn't be. They should be accessed via game instance. By `Model.getRoom().getGame()` you have the game instance and then you can access non static fields.